### PR TITLE
Tooltip tests and bug fix

### PR DESCRIPTION
**Pull request by stdavis**
Ported tests to intern

Fixed a bug when using `container: body`

### CODE REVIEW

**Comment by tomwayson** [comment]
@stdavis so sorry I've been sitting on this. Been busy w/ stuff of other repos and this has totally slipped off my radar - I'll try to get to it by tomorrow.

**Comment by stdavis** [comment]
No worries.

On Mon, Apr 20, 2015 at 8:20 PM, Tom Wayson <notifications@github.com>
wrote:

> @stdavis <https://github.com/stdavis> so sorry I've been sitting on this.
> Been busy w/ stuff of other repos and this has totally slipped off my radar
> - I'll try to get to it by tomorrow.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/xsokev/Dojo-Bootstrap/pull/154#issuecomment-94616086>.
>



-- 
*Scott Davis*
Utah AGRC <http://gis.utah.gov/developer/>
stdavis@utah.gov
@ScottAGRC <https://twitter.com/ScottAGRC>

**Comment by stdavis** [comment]
Anything that I can do to help on this? Not trying to pressure just thought that I would give this a bump. I'd be interested in helping wire this project up to Travis if someone wants to switch it on at travis-ci.org. I could also hook it up to sauce if you are interested. Might make PR's easier to digest.

**Comment by tomwayson** [comment]
sorry @stdavis - i've been slammed. I keep thinking I'll get to this at the end of each day, but I never do. If I don't get to it today, I'll take my name off.

Re: travis - I thought we already had an open issue for that, but I guess it was just mentioned in comments. That would be awesome. I think as a contributor I can wire this up to my travis acct, but maybe I have to be the owner. Not sure, but if you want to open issue and/or PR, I can try it.

Re: sauce labs - bonus!

**Comment by stdavis** [comment]
No worries. Totally understand. I may do a bit of work on Travis/Sauce
later today.

On Fri, Apr 24, 2015 at 10:56 AM, Tom Wayson <notifications@github.com>
wrote:

> sorry @stdavis <https://github.com/stdavis> - i've been slammed. I keep
> thinking I'll get to this at the end of each day, but I never do. If I
> don't get to it today, I'll take my name off.
>
> Re: travis - I thought we already had an open issue for that, but I guess
> it was just mentioned in comments. That would be awesome. I think as a
> contributor I can wire this up to my travis acct, but maybe I have to be
> the owner. Not sure, but if you want to open issue and/or PR, I can try it.
>
> Re: sauce labs - bonus!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/xsokev/Dojo-Bootstrap/pull/154#issuecomment-95991367>.
>



-- 
*Scott Davis*
Utah AGRC <http://gis.utah.gov/developer/>
stdavis@utah.gov
@ScottAGRC <https://twitter.com/ScottAGRC>

File: tests/assets/tests.css [hunk 1]
```diff
@@ -1,4 +1,4 @@
-@import url("//netdna.bootstrapcdn.com/bootstrap/3.1.1/css/bootstrap.min.css");
+@import url("//netdna.bootstrapcdn.com/bootstrap/3.3.4/css/bootstrap.min.css");
```
Comment (tomwayson) [inline tests/assets/tests.css]: Nice to see the tests running against BS 3.3.4. I never bumped it up b/c I felt like we never got through 3.1.1

**Comment by tomwayson** [comment]
Intern tests pass (though take a couple minutes??? to run - yay Intern). Test pages look good. Thanks againg

**Pull request merged**